config/org.yaml

```diff
@@ -433,6 +434,7 @@ orgs:
           - dwindsor
           - FedeDP
           - EXONER4TED
+          - Lowaiz
```

Comment:
This change should end up in a follow-up PR, because we first must:
* add you to the falcosecurity org
* you need to accept the invite via mail
* then we will merge the driverkit PR
* finally, you will need to add yourself here